app/src/main/java/org/oppia/app/profile/AdminPinActivity.kt

```diff
@@ -0,0 +1,36 @@
+package org.oppia.app.profile
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+const val KEY_ADMIN_PIN_PROFILE_ID = "ADMIN_PIN_PROFILE_ID"
+
+/** Activity that sets the admin's PIN. */
+class AdminPinActivity : InjectableAppCompatActivity() {
+  @Inject
+  lateinit var adminPinActivityPresenter: AdminPinActivityPresenter
+
+  companion object {
+    fun createAdminPinActivity(context: Context, profileId: Int, color: Int): Intent {
```

Comment:
As per this comment (https://github.com/oppia/oppia-android/pull/539/files#r356367653) the `color` name will get updated here also.